browser/brave_adaptive_captcha/brave_adaptive_captcha_service_factory.cc

```diff
@@ -30,7 +32,17 @@ class CaptchaDelegate
 
   bool ShowScheduledCaptcha(const std::string& payment_id,
                             const std::string& captcha_id) override {
-    return brave_rewards::ShowRewardsPanel(context_, true);
+    auto* profile = Profile::FromBrowserContext(context_);
+    auto* browser = chrome::FindTabbedBrowser(profile, false);
+    if (!browser) {
+      return false;
```

Comment:
Should we open a new browser window in this case? Seems like an impossible situation if we're triggering it from a tooltip so actually if there's nothing that should trigger this in the background maybe we should DCHECK here?

Reply:
I'm not comfortable changing the behavior in this PR, but perhaps we should create an issue for Adaptive Captcha?